eva_sub_cli/validators/validator.py

```diff
@@ -54,7 +56,10 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.sub_config.backup()
         self.sub_config.write()
-
+    @staticmethod
+    def _quiet_commands(command_description, command, **kwargs):
```

Comment:
The plural "commands" seems weird to me, maybe:
```suggestion
    def _run_quiet_command(command_description, command, **kwargs):
```